clients/admin-ui/src/pages/login.tsx

```diff
@@ -83,26 +77,19 @@ const useLogin = () => {
   };
 
   const getRedirectRoute = () => {
-    if (!token) {
+    if (!token || isSystemsLoading) {
```

Comment:
this was a weird bug: I think it was masked earlier since the config wizard flag was always off. for some reason it only shows up when running the app via `cy:start` instead of regular ol' `npm run dev`, too, at least locally. not sure what that would be, but this seems to work for both cases